src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (x) => {
+    return (y) => {
+      counter++;
+
+      if (counter <= 3 || counter % 2 !== 0) {
+        return x + y;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
Else is redundant here
```suggestion
      }
      
      return 'Bzzz... Error!';
```